lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php

```diff
@@ -49,25 +49,26 @@
 
   <section class="container -padded">
     <div class="wrapper">
-      <div class="container__body -narrow">
-        <?php if (!empty($facts)): ?>
+      <?php if (!empty($facts)): ?>
+        <div class="container__body -narrow">
           <h2 class="inline--alt-color"><?php print $facts_header; ?></h2>
           <ul>
             <?php foreach ($facts['facts'] as $fact): ?>
               <li><?php print $fact['fact']; ?> <sup><?php print $fact['footnotes']; ?></sup></li>
             <?php endforeach; ?>
           </ul>
-
-          <section class="sources">
+        </div>
+        <div class="container__body -narrow">
```

Comment:
So why does the "sources" block need to be enclosed in a separate `.container__body` as well if it relates to the information above? Can't it just be placed inside the `.container__body` for the facts list with some margin above it.